### ISSUE TITLE
Production releases contain dev javascript bundles

### ISSUE THREAD

**Issue by AlanFoster** [Thinkmill/react-native-schemes-manager#11]
### Steps to reproduce the behavior

Build a production version of the react-native application, and output the value of `__DEV__`

### Expected behavior

The value of `__DEV__` is set to false, and all dev tools are removed - i.e. prop type warnings etc.

### Actual behavior

The value of `__DEV__` is set to true, even though it is a production build.

### Reason

The current dynamic case statement has syntax issues:

```
Showing All Issues
../node_modules/react-native-schemes-manager/lib/react-native-xcode.sh: eval: line 88: syntax error near unexpected token `)'
../node_modules/react-native-schemes-manager/lib/react-native-xcode.sh: eval: line 88: `  )'
```

Subsequently the status code is ignored, meaning `$DEV` is not set.
When `$DEV` is not set, the default bundling mechanism for react-native is `true`.

Therefore when using this plugin and requesting a production build, we are still given the dev build.

I'll put a PR up for this

**Comment by blargity** [comment]
Well spotted, thank you!

**Comment by blargity** [comment]
This was resolved in #12, and released in `v1.0.0-beta.12`.
